src/api/db/migrate/20161124135426_drop_obsolete_table.rb

```diff
@@ -0,0 +1,10 @@
+class DropObsoleteTable < ActiveRecord::Migration
+  def up
+    drop_table :bs_request_histories
+  rescue ActiveRecord::StatementInvalid
+    # just drop in case it exists, it is not used since two years
```

Comment:
The old migration is supposed to remove it, but it seems not to work always. 

Just new installations should have that table. Anyway, better safe than sorry here and avoid a migration failure. Since it is worse to have an aborted migration than one not used table too much.